### DISCUSSION TITLE
Highlight where the player failed in DefaultSongProgress/ArgonSongProgress

### DISCUSSION THREAD

**Discussion by Nowaaru** [ppy/osu#23298] [Ideas] [answered]
It would be nice to have a little red arrow or pill over the progress bar, indicative of where the user failed. Or, will we get some kind of really cool awesome osu! skinning scripting language that would allow us to do it ourselves? Either way, a bit'a support would be awesome.

My reason behind it is that I frequently find myself scrubbin' around the timeline trying to find that really cool part that I hit but I can't find exactly where it is. I've spent an embarrassing amount of time, over several times, suffering from that. I might just be.. stupid.

<sub>searches: `replay fail`, `fail`, `replay`</sub>
<sub><sub><sub>sorry for disappearing on my last post i realized i was dumb and i got too embarrassed to respond :(</sub></sub></sub>

**Comment by Gabixel** [comment, marked as answer]
This is likely covered in #5681. I mainly posted that to have some sort of "history" during the re-/play ([and something similar to osu!stream's design since the Argon skin implementation](https://github.com/ppy/osu/discussions/22453)), but I would also like the idea to find where you failed in the map (but all of this probably requires to store something in the replay, or make the game seek/load through before you if we want that information at the beginning of a replay).
Maybe it can also be considered for [this suggestion](https://github.com/ppy/osu/issues/22298)
